src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/SynthesizedClosureEnvironment.cs

```diff
@@ -89,13 +98,20 @@ private static void AssertIsClosureScopeSyntax(SyntaxNode syntaxOpt)
 
         public override ImmutableArray<Symbol> GetMembers()
         {
-            var members = base.GetMembers();
+            var builder = ArrayBuilder<Symbol>.GetInstance();
+            builder.AddRange(base.GetMembers());
```

Comment:
We might not want to change the order of members to avoid unnecessary changes in the emitted metadata.

I'd move AddRange to the end of the method.